### PR TITLE
adding server.js and a resource file

### PR DESCRIPTION
**Pull request by PhiTarte**

### CODE REVIEW

**Review by dadtmt — CHANGES_REQUESTED** [review]
resource.js is not used yet in the project, make it a  module and import it in server.js
in server.js you are reffering to bootstrap.min.css but the file is not included in your project use bootstrap cdn instead

**Review by dadtmt — CHANGES_REQUESTED** [review]
missing the meta tag viewport check bootstrap doc

File: server.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <script src="contenu.js"></script>
+  </body>
+</html>`
+
+// i will do something when receiving the homepage request
+app.get('/', (req, res) => {
+  const HtmlAjout = `
```
Comment (dadtmt) [inline server.js]: use english and camelCase notation

File: server.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  <div>HTML homepage</div>
+  `
+  res.send(html(HtmlAjout+htmlFooter))
+  res.end()
+})
+
```
Comment (dadtmt) [inline server.js]: why so many roads ? did you understand what is a SPA ? did you check how does the Pirates exemple work ? Did you plan to reload every time you hit a road ?

**Review by dadtmt — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (El-Loco-Pinguino) [reply server.js]: We have many routes because we plan to create many sections (cf. our user stories). Actually, we're not really sure how do routes work.

Comment (dadtmt) [reply server.js]: and now ?

File: server.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+const sqlite = require('sqlite')
+const express = require('express')
+const bodyParser = require('body-parser')
+<<<<<<< HEAD
```
Comment (dadtmt) [inline server.js]: oups test your code after merging :(
this happense when you fix conflict and commit without saving the files

**Review by dadtmt — CHANGES_REQUESTED** [review]

File: server.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+</html>`
+
+app.get('/wilders', (req, res) => {
+  //renvoyer un arrayjson de  wilders
+  res.json(wilders)
+  res.end()
+})
+
+// i'm sending the skeleton of the page (when i'm receiving any of the routes)
+app.get('*', (req, res) => {
```
Comment (dadtmt) [inline server.js:46]: duplicated code

**Review by dadtmt — CHANGES_REQUESTED** [review]

File: public/app.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    </div>`)
+
+  ),
+    
+
+    '*': () => render('<h1>Not Found</h1>')
+}
+
+
+const route = pathname => {
```
Comment (dadtmt) [inline public/app.js:36]: look at a more readable way to do this in nodejs vanilla

**Review by dadtmt — APPROVED** [review]

**Pull request merged**